Controllers/NoteController.cs

```diff
@@ -0,0 +1,53 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Taskmanager.Controllers.InputModels;
+using Taskmanager.Controllers.ViewModels;
+using Taskmanager.Data.Entities;
+using Taskmanager.Services.Interfaces;
+
+namespace Taskmanager.Controllers
+{
+    [ApiController]
+    [Route("api/notes")]
+    public class NoteController : ControllerBase
+    {
+        private readonly INoteService _noteService;
+
+        public NoteController(INoteService noteService)
+        {
+            _noteService = noteService;
+        }
+
+        [HttpGet("{userId}/{todolistId}/{todoitemId}")]
```

Comment:
You don't need combination of userId + todolistId + todoitemId to identify note, note can be retrieved just by unique todoitemId and it will be enough to identify unique db entity